java/src/test/java/ai/rapids/cudf/ScalarTest.java

```diff
@@ -287,4 +306,95 @@ public void testList() {
       }
     }
   }
+
+  @Test
+  public void testStruct() {
+    try (ColumnVector col0 = ColumnVector.fromInts(1);
+         ColumnVector col1 = ColumnVector.fromBoxedDoubles(1.2);
+         ColumnVector col2 = ColumnVector.fromStrings("a");
+         ColumnVector col3 = ColumnVector.fromDecimals(BigDecimal.TEN);
+         ColumnVector col4 = ColumnVector.daysFromInts(10);
+         ColumnVector col5 = ColumnVector.durationSecondsFromLongs(12345L);
+         Scalar s = Scalar.structFromColumnViews(col0, col1, col2, col3, col4, col5, col0, col1)) {
```

Comment:
nit: Add checks for scalar type, and validity.

Reply:
Fixed.